titus-api/src/main/java/com/netflix/titus/api/jobmanager/model/job/sanitizer/JobConfiguration.java

```diff
@@ -70,7 +70,7 @@
      * An upper bound on memory (megabytes) a single container may allocate. The actual limit may be lower, as it also depends
      * on instance types available in a tier.
      */
-    @DefaultValue("472000")
+    @DefaultValue("760000")
```

Comment:
should we leave some slack for executor operations + spike during coredumps?

Reply:
I left a couple of G but I could reduce it to be 750000 so there is > 10GB buffer.

Reply:
Yes, `750000` might be better